tests/gql_behave/run.py

```diff
@@ -92,9 +95,27 @@ def add_argument(option, **kwargs):
     behave_args.extend(["--test-suite", parsed_args.test_suite])
     behave_args.extend(["--test-directory", test_directory])
 
+    # Setup database
+    if parsed_args.storage_mode == "in_memory":
```

Comment:
Add storage mode flag will be merged today -> no need to introduce setup queries, we can just start Memgraph with any storage mode.